Source/WebCore/Modules/ShapeDetection/Implementation/Cocoa/VisionUtilities.mm

```diff
@@ -80,15 +80,17 @@ void configureRequestToUseCPUOrGPU(VNRequest *request)
     for (VNComputeStage computeStage in supportedComputeStageDevices) {
         bool set = false;
         for (id<MLComputeDeviceProtocol> device in supportedComputeStageDevices[computeStage]) {
-            if ([device isKindOfClass:PAL::getMLGPUComputeDeviceClass()]) {
+            RetainPtr mlgpuComputeDeviceClass = PAL::getMLGPUComputeDeviceClass();
```

Comment:
The changes to this file are no longer needed. We've fixed this recently in the last couple days.